server/src/Logger.ts

```diff
@@ -0,0 +1,22 @@
+export enum LogLevel {
+  Debug = '__LogLevelDebug',
+}
+
+export class Logger {
+  static prepareLogger(workspaceFolder: string): void {
+    const id = workspaceFolder.substring(workspaceFolder.lastIndexOf('/') + 1);
+
+    const old = console.log;
+    console.log = (...args): void => {
+      const count = Array.prototype.unshift.call(args, `${id} ${new Date().toISOString()}: `);
+      if (count >= 3 && args[2] === LogLevel.Debug) {
```

Comment:
If `count >= 3` then `args[2]` is not `LogLevel` ?

Reply:
In this case it is not level:
`console.log('a','b')`